operations/network/network_management_install_guide/aruba/initializing_canu.md

```diff
@@ -0,0 +1,35 @@
+# Initializing Canu
+
+To help creating the switch configuration. CANU can automatically parse CSI output or the Shasta SLS API for switch IPv4 addresses.
+
+### CSI Input
+
+- In order to parse CSI output, use the `--csi-folder FOLDER` flag to pass in the folder where the _sls_input_file.json_ file is located.
+
+The _sls_input_file.json_ file is generally stored in one of two places depending on how far the system is in the install process.
+
+Early in the install process, when running off of the LiveCD the _sls_input_file.json_ file is normally found in the the directory `/var/www/ephemeral/prep/SYSTEMNAME/`
```

Comment:
For consistency, use a single method of formatting filenames and paths. The convention for the rest of the HPE Cray EX product stream docs is to use single backticks, which are also used for commands.